packages/larva/scripts/prod.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Run a production ready build in a Larva project.
+ *
+ * ## OPTIONS
+ *
+ * [--parse]
+ * : Run the Twig-to-PHP parser.
+ *
+ * [--parser]
+ * : Alias for --parse.
+ *
+ * [--json]
+ * : Run the write-json script.
+ *
+ * [--write-json]
+ * : Run the write-json script.
+ *
+ * [--css]
+ * : Build CSS assets.
+ *
+ * [--scss]
+ * : Alias for --css.
+ *
+ * [--js]
+ * : Build JavaScript assets.
+ *
+ * [--svg]
+ * : Build SVG assets.
+ *
+ * ## EXAMPLES
+ *
+ *     # Run a full build
+ *     $ npx @penskemediacorp/larva prod
+ *
+ *     # Only run the write-json script.
+ *     $ npx @penskemediacorp/larva lint --write-json
```

Comment:
Is this part of the `lint` script? write-json is not a linter

Reply:
Nope, just the `prod` replacement.

Reply:
Okay - I think I'm not understanding `npx @penskemediacorp/larva lint --write-json` in this context

Reply:
Is it supposed to be `npx @penskemediacorp/larva prod --write-json`, and same for others in this comment?

Reply:
Oh, yeah, that's just a typo. I'll address.